src/basic/log.c

```diff
@@ -1458,7 +1458,12 @@ void log_settle_target(void) {
 }
 
 int log_get_max_level(void) {
-        return log_max_level;
+        int max = 0;
```

Comment:
please use LOG_NULL for this

Reply:
(i.e. LOG_NULL is the log level for disabling the whole thing entirely, see its definition. it's effectively -1)